devices/philips.js

```diff
@@ -1636,9 +1672,16 @@ module.exports = [
         model: '4080248U9',
         vendor: 'Philips',
         description: 'Hue White and color ambiance Signe floor light',
-        toZigbee: [tzLocal.gradient_scene, ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).toZigbee],
-        exposes: [exposes.enum('gradient_scene', ea.SET, Object.keys(gradientScenes)),
-            ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).exposes],
+        toZigbee: [
+            tzLocal.gradient_scene,
+            tzLocal.philips_hue_multicolor({reverse: true}),
+            ...hueExtend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}).toZigbee,
+        ],
+        exposes: [
+            exposes.enum('gradient_scene', ea.SET, Object.keys(gradientScenes)),
+            exposes.list('colors', ea.SET, exposes.text('color', ea.SET)),
```

Comment:
Can you add a description to the `color`? (required format example). Something like:

`exposes.text('color', ea.SET)` -> `exposes.text('color', ea.SET).withDescription('Color in hex e.g. #DEEB34')`

Reply:
Yes, will do! I figured out that it probably makes more sense to use X/Y colors instead of RGB, as that's what the device is using, so I'm rewriting this now. I'll ping you once it's ready for another review.